reddit.js

```diff
@@ -1,5 +1,38 @@
 var reddit = require("./data.json")
+// console.log(reddit["data"]);
 
 // Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+// forEach - cosole.log all titles
+reddit.data.children.forEach(function (titles) {
+	console.log(titles.data.title);
+});
+
+// map - return an array of permalinks, append http://reddit.com to each so they contain full http:// path to reddit
+var permalinks = reddit.data.children.map(function (link) {
```

Comment:
Same here as with the above, your parameter in your iterative function should reflect what's being iterated over. In this case, it's a *child* of ```children```.